### ISSUE TITLE
Pairs of device_id and don't work

### ISSUE THREAD

**Issue by semsom** [coder-fly/douyin_device_register#10]
At the moment pairs doents work for tiktok I think it's all about the template of device. Does anyone know how this can be changed? I just don't understand where exactly the request body is encrypted e.g. "( { fingerprint = ""; header = { access = WIFI; aid = 1128; "app_language" = zh; "app_name" = aweme; "app_region" = RU; "app_version" = "8.7.1"; carrier = "\U4e2d\U56fd\U79fb\U52a8"; channel = AppStore; custom = { "app_language" = zh; "app_region" = CN; "build_number" = 87100; "earphone_status" = on; }; "device_id" = ; "device_model" = "iPhone X"; "display_name" = "\U6296\U97f3\U77ed\U89c6\U9891"; idfa = "E3D93D3-U747-R394-E2033-HF383J3984JE"; "install_id" = ; "is_jailbroken" = 0; "is_upgrade_user" = 1; language = zh; mc = "00:00:00:00:00:00"; "mcc_mnc" = ""; openudid = 9234923948d9392934dkk3939935d93919r3a3s3; os = iOS; "os_version" = "12.1"; package = "com.ss.iphone.ugc.Aweme"; region = CN; resolution = "1024*768"; "sdk_version" = 0011; timezone = 1; "tz_name" = "Asia/Shanghai"; "tz_offset" = 99000; "user_agent" = "Aweme 8.7.1 rv:87100 (iPhone; iPhone OS 12.1; zh_CN) Cronet"; "vendor_id" = "6J3DJD34-3DA4-R3KD-DS33-739394839384"; }; "magic_tag" = "ss_app_log"; } "
Or does anyone know how to update something like this in the jar archive. Thank you.

**Comment by liamthedux124** [comment]
I could say it needs activation via either applog request or xlog idk which one exactly 

anyway any progress

**Comment by Ozussh** [comment]
> I could say it needs activation via either applog request or xlog idk which one exactly
> 
> anyway any progress

log every minute whenever install id has been used

**Comment by uimygod** [comment]
> > I could say it needs activation via either applog request or xlog idk which one exactly
> > anyway any progress
> 
> log every minute whenever install id has been used

I dont quite get u maybe rephrase it

**Comment by xtekky** [comment]
@semsom this is only applog, you have to activate these devices via xlog to actually use them
